tests/integration/test_script_runner.py

```diff
@@ -549,18 +549,24 @@ class IntegrationTestFixture:
         data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
         extra_backend_dependencies=BackendDependencies.BIGQUERY,
     ),
+    # IntegrationTestFixture(
+    #     name="split_data_on_datetime_bigquery",
+    #     user_flow_script="tests/integration/db/test_sql_data_splitting.py",
+    #     data_context_dir="tests/integration/fixtures/no_datasources/great_expectations",
+    #     data_dir="tests/test_sets/taxi_yellow_tripdata_samples/",
+    #     util_script="tests/test_utils.py",
+    #     other_files=(
+    #         (
+    #             "tests/integration/fixtures/split_data/bigquery_connection_string.yml",
+    #             "connection_string.yml",
+    #         ),
+    #     ),
+    #     extra_backend_dependencies=BackendDependencies.BIGQUERY,
+    # ),
```

Comment:
Currently this Splitter test is failing in BigQuery and also takes a long time (~10 min) to run. Would like  to consider moving to `azure-pipelines-cloud-db-integration.yml` pipeline